ida-customizations/.vuepress/config.js

```diff
@@ -89,31 +89,31 @@ module.exports = {
               children: [
                 {
                   title: "Getting Started",
-                  path: "/ida-course/LPs/week-0/",
+                  path: "ida-customizations/ida-course/LPs/week-0/",
```

Comment:
```suggestion
                  path: "ida-course/LPs/week-0/",
```
@coldice Should it be like this instead?